docs/reference/requirement-sets/excel-api-requirement-sets.md

```diff
@@ -52,11 +52,200 @@ For more information about versions, build numbers, and Office Online Server, se
+||[left](/javascript/api/excel/excel.range#left)|Returns the distance in points, for 100% zoom, from left edge of the worksheet to left edge of the range. Read-only.|
+||[top](/javascript/api/excel/excel.range#top)|Returns the distance in points, for 100% zoom, from top edge of the worksheet to top edge of the range. Read-only.|
+||[width](/javascript/api/excel/excel.range#width)|Returns the distance in points, for 100% zoom, from left edge of the range to right edge of the range. Read-only.|
+|[Shape](/javascript/api/excel/excel.shape)|[copyTo(destinationSheet?: Worksheet \| string)](/javascript/api/excel/excel.shape#copyto-destinationsheet-)|Copies and pastes a Shape object.|
+||[placement](/javascript/api/excel/excel.shape#placement)|Represents how the object is attached to the cells below it.|
+|[ShapeCollection](/javascript/api/excel/excel.shapecollection)|[addSvg(xml: string)](/javascript/api/excel/excel.shapecollection#addsvg-xml-)|Creates a scalable vector graphic (SVG) from an XML string and adds it to the worksheet. Returns a Shape object that represents the new image.|
+|[Slicer](/javascript/api/excel/excel.slicer)|[caption](/javascript/api/excel/excel.slicer#caption)|Represents the caption of slicer.|
+||[clearFilters()](/javascript/api/excel/excel.slicer#clearfilters--)|Clears all the filters currently applied on the slicer.|
+||[delete()](/javascript/api/excel/excel.slicer#delete--)|Deletes the slicer.|
+||[getSelectedItems()](/javascript/api/excel/excel.slicer#getselecteditems--)|Returns an array of selected items' keys. Read-only.|
+||[height](/javascript/api/excel/excel.slicer#height)|Represents the height, in points, of the slicer.|
+||[left](/javascript/api/excel/excel.slicer#left)|Represents the distance, in points, from the left side of the slicer to the left of the worksheet.|
+||[name](/javascript/api/excel/excel.slicer#name)|Represents the name of slicer.|
+||[nameInFormula](/javascript/api/excel/excel.slicer#nameinformula)|Represents the name used in the formula.|
+||[id](/javascript/api/excel/excel.slicer#id)|Represents the unique id of slicer. Read-only.|
+||[isFilterCleared](/javascript/api/excel/excel.slicer#isfiltercleared)|True if all filters currently applied on the slicer is cleared.|
+||[slicerItems](/javascript/api/excel/excel.slicer#sliceritems)|Represents the collection of SlicerItems that are part of the slicer. Read-only.|
+||[worksheet](/javascript/api/excel/excel.slicer#worksheet)|Represents the worksheet containing the slicer. Read-only.|
+||[selectItems(items?: string[])](/javascript/api/excel/excel.slicer#selectitems-items-)|Select slicer items based on their keys. Previous selection will be cleared.|
+||[sortBy](/javascript/api/excel/excel.slicer#sortby)|Represents the sort order of the items in the slicer. Possible values are: DataSourceOrder, Ascending, Descending.|
```

Comment:
Is there a link to the enums for possible values?

Reply:
There would be in the sortBy section of the ref doc, if not for the [usual API Extractor bug](https://github.com/Microsoft/web-build-tools/issues/867). That said, the values are listed out right there. I'm not anyone needs a link.